pootle/static/js/browser/components/BrowserTable.js

```diff
@@ -131,10 +131,16 @@ const BrowserTable = React.createClass({
     this.setState({ reverseSort, sortColumn });
   },
 
+  isItemDisabled(item) {
```

Comment:
Nit: This doesn't depend on anything related to the class and personally I'd make it a regular function. However, use your best judgment.